src/features/content-explorer/content-explorer/ContentExplorerInfoNotice.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { InfoBadge } from '@box/blueprint-web-assets/icons/Line';
+
+import InlineNotice from '../../../components/inline-notice';
+
+const ContentExplorerInfoNotice = ({ infoNoticeText }) => (
+    <InlineNotice type="info" className="content-explorer-info-notice">
```

Comment:
should this use the InlineNotice from blueprint?

Reply:
Right, updated. Now the whole implementation is simpler